do.sh

```diff
@@ -17,4 +21,17 @@ if ! grep -q "set editing-mode vi" "${HOME}/.inputrc" 2>/dev/null; then
 fi
 
 
+echo "Configuring iptables firewall..."
+iptables -F
+iptables -X
+iptables -P INPUT DROP
+iptables -P FORWARD DROP
+iptables -P OUTPUT ACCEPT
+iptables -A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+iptables -A INPUT -i lo -j ACCEPT
+iptables -A INPUT -p tcp --dport "${SSH_PORT}" -j ACCEPT
+iptables -A INPUT -p tcp --dport 80 -j ACCEPT
```

Comment:
Limit the source port too: `--sport 1024:65535`.

Reply:
Done